### DISCUSSION TITLE
How to add custom middleware

### DISCUSSION THREAD

**Discussion by fan-tastic-z** [loco-rs/loco#378] [General]
How to add custom middleware，and are considering adding intermediate instructions to the document in the future?

**Comment by kaplanelad** [comment]
@fan-tastic-z, you can use `after_routes` hook. Shuttle blog wrote a short guid [here](https://www.shuttle.rs/blog/2023/12/28/using-loco-rust-rails#middleware-in-loco).


Do you want to open a pull request and add it to loco.rs documentation?

**Reply by fan-tastic-z** [reply]
thanks，I mean whether to consider how to add custom middleware to the loco.rs document
